dynamic_landing_page/script.js

```diff
@@ -0,0 +1,103 @@
+(function() {
+const time = document.getElementById("time"),
+greeting = document.getElementById("greeting"),
+name = document.getElementById("name"),
+focus = document.getElementById("focus");
+
+// Show Data
+const showFullDate = () => {
+let data = new Date(),
+hours = data.getHours(),
+minutes = data.getMinutes(),
+seconds = data.getSeconds();
+hours = hours % 12 || 12;
+minutes = minutes<10 ? "0" + minutes : minutes;
+seconds = seconds<10 ? "0" + seconds : seconds;
+time.innerText = `${hours}:${minutes}:${seconds} ${amPm()}`;
+setTimeout(showFullDate, 1000)
+};
+
+const amPm = () => {
+    let data = new Date(),
+    hours = data.getHours();
+    return hours <= 12 ? "AM":"PM";
+}
+showFullDate();
+
+//Change background
+const changeBackground = () =>{
```

Comment:
Some code inside the function is duplicated, like `body.style.backgroundRepeat = "no-repeat";`
At the begining of the function you can determing the  daytime, like:
```
const isNight = hours => (hours>0 && hours <5) || hours>=22;
const isMorning =hours =>  hours>=5 && hours <12;
const isAfternoon = hours => hours>=12 && hours <18;
let datetime = isNight(hours) ? 'night' : 
                         isMorning(hours) ? 'morning' :
                         isAfternoon(hours) ? 'afternoon' :
                         'evening';
    greeting.innerText = `Good ${daytime}, `
    body.style.background = `url(./pictures/${daytime}.jpg)`;
    body.style.backgroundRepeat = "no-repeat";
    body.style.backgroundSize = "cover";
```